### ISSUE TITLE
Create new tag/release

### ISSUE THREAD

**Issue by samm-git** [rawstudio/rawstudio#88]
Dear authors,

Latest release was done in 2017, but according to the git i see that there are many changes since then, including gtk3 support.

Would it be possible to release a new version, or, at least, create some tags, to make maintainers and users life easier? 

Thank you

**Comment by alerque** [comment]
See #53. Several of us have tried off and on to get the ball rolling again but so for nobody has been given a green light to run with anything. A little bit of progress was made on PR review, but it is all still dependent on the original developer to actually merge anything and prep releases.

**Comment by samm-git** [comment]
@abrander any chance to create a release (we can call it beta, rc, whatever?). I am maintainer of the FreeBSD port and it would be really nice to have something new, as backporting all old patches is not an efficient task.

**Comment by abrander** [comment]
I would love to tag a new release - preview, beta, whatever. What needs to happen? I assume it's not as simple as updating `AC_INIT` and running `git tag` on the latest commit in master.

@alerque, I would love the help. Right now, we have two rather old PR needing review, and probably a few issues that can be easily triaged.

**Comment by samm-git** [comment]
Thank you for taking that :) I think it should be as easy as that. You already have 2.1 version in the `AC_INIT`, so probably you can create new tag v2.1 and then update it to 2.2 or whatever will be next release :) I would not really bother too much about rc/beta/preview, IMO its easier to just drop new tag once done. I can also try to contribute some CI scripts to check that things are still buildable.

**Comment by alerque** [comment]
Doing a slightly sloppy release could be "as simple as that", but yes I would do a little more homework on this end to ease adoption on the other end. That includes some release docs that help distro packagers know how to adjust, pinning some dependencies, etc.

I'll help out as I have time — *on the condition I or somebody else gets some access first*. I've been suggesting for years that we could help and both I and others (notably @sergiomb2) have *tried* to do issue and PR triage as well as sent contributions but we keep getting stifled on progress because we have to wait on you to actually handle issues. It's not that you haven't *eventually* gotten back to us, but in a couple cases it took months and years. Even if you happen to be available right this moment I'm just a bit skittish about putting time into this now since I'm not confident we'll get to the end of the tunnel. Add myself (and perhaps Sérgio) with collaborator status on this repo and enough privs to mess with CI and releases as well as issue/pr management and then I'll do my best to jump in. If my track record with maintaining other projects (including many owned by other folks) and distro packaging history isn't enough to trust me with that then  ... well then I'll still be around but this may not move up from the bottom of my todo list. It's just too demotivating. That's not a personal jab at all, just noting how it feels as a would-be contributor on a project that is badly broken/stale and has been for years yet the contribution workflow is gate-kept/slow-balled such that every distro that packages it has at least at some point had to patch or package forks.

**Comment by sergiomb2** [comment]
TBH I haven't tested rawstudio for a while , but I notice rawstudio depends on GConf2 which is very old and we want remove it from Fedora . Also doesn't work with wayland ... .i.e migration of GCconf2 also should be study and check running rawstudio with wayland

**Comment by abrander** [comment]
Cutting a Rawstudio release requires more mental energy than I have available right now.

Here's my suggestion: I will give the necessary permissions to @alerque and @sergiomb2 if they want to co-maintain the Rawstudio project. The first order of business could be discussing what's needed for the next release to happen.

@anderskvist Maybe you have a few thoughts too?

**Comment by sergiomb2** [comment]
> The first order of business could be discussing what's needed for the next release to happen 
- use libraw
- drop gconf2 , and IIRC use gsettings
- fix crash when lunch in wayland 

- enable geo location  (optional)

**Comment by esbante** [comment]
Any news regarding the situation here? Would really like to see a new release as this program is just too good (there really is no other raw converter that is as fast and nice to work with as this one) to fade away from existence.
